### ISSUE TITLE
Improper noctua-models checkout prevents pipeline from running normally

### ISSUE THREAD

**Issue by kltm** [geneontology/go-site#2336]
The go pipelines are now regularly failing with errors like:

```
07:36:32  error: RPC failed; curl 92 HTTP/2 stream 0 was not closed cleanly: CANCEL (err 8)
07:36:32  error: 6349 bytes of body are still expected
07:36:32  fetch-pack: unexpected disconnect while reading sideband packet
07:36:32  fatal: early EOF
07:36:32  fatal: fetch-pack: invalid index-pack output
07:36:33  make: *** [Makefile:166: target/noctua-models] Error 128
07:36:33  make: *** Waiting for unfinished jobs....
```

The background for this is that we're using GitHub as file storage. Initially, we used (versioning?) S3, but changed to GH some time ago for better visibility and tracking. As the repo has grown enormously since then, we have understood that we'd eventually run into the limits of using GH as the backing store. This would seem to be the initial stages of that.

Basically, what seems to be happening, is that checking out the _entire_ deep repo causes GH to hang up; just like the ontology, we need to get into the practice of checking out the absolute minimum we can get away with. It is understood that this is just kicking the can down the road. (Also noting that switching back to S3 is likely not _that_ big a deal, but would require the update of a few jobs and scripts, as well as losing diffing ability: heads-up to @vanaukenk for that eventuality at an unknown point in the future.)

**Comment by kltm** [comment]
Tagging @mugitty

**Comment by kltm** [comment]
Restarting `snapshot` with fix.

**Comment by mugitty** [comment]
Thanks @kltm . I have ported the changes to https://github.com/geneontology/go-site/tree/issue-2246-gorule-0000001

**Comment by kltm** [comment]
@mugitty Hopefully this will be successful! (Local experimentation is positive so far.)

**Comment by kltm** [comment]
@pgaudet Apologies, I'm still having a little problem with this. I think I have the fix in and I'm rerunning now.

**Comment by kltm** [comment]
Looks like we finally got on to the second stage.

**Comment by kltm** [comment]
Okay, now have it in `snapshot-post-fail`

**Comment by kltm** [comment]
I believe this has been cleared.
